optimizers/scalar_opts.py

```diff
@@ -185,3 +186,122 @@ def lion_update_foreach(
     # X = X - lr * U
     torch._foreach_mul_(U, lr)
     torch._foreach_sub_(X, U)
+
+
+class AdamW(torch.optim.Optimizer):
```

Comment:
Remove AdamW and Lion optimizer classes. The functions should be tested directly.